tools/schemapi/schemapi.py

```diff
@@ -562,11 +562,17 @@ def to_dict(self, validate=True, ignore=None, context=None):
             try:
                 self.validate(result)
             except jsonschema.ValidationError as err:
-                raise SchemaValidationError(self, err)
+                raise SchemaValidationError(self, err) from err
         return result
 
     def to_json(
-        self, validate=True, ignore=[], context={}, indent=2, sort_keys=True, **kwargs
+        self,
+        validate=True,
+        ignore=None,
```

Comment:
Same remark here about docstring as in altair/utils/schemapi.py

Reply:
Done